### ISSUE TITLE
Notification Error

### ISSUE THREAD

**Issue by JavaNameConvention** [jariz/Akku#89]
Hello, 
I am currently using Akku 0.1.0 beta 11, running on MacOS big sur version 11.1 on MacBook Pro 2020 13inch . I am experiencing 2 issues. First, after I install the helper, I have to restart before the Akku battery menu actually opens. This is a small issue, but its still an issue. My second issue is a bit bigger. When I toggle notifications to on, and my Bluetooth device goes under the notification percentage, notifications start being sent in every second. So, for example, when my Bluetooth earbuds(K010 - L) went under 20%, it gave me 3 notifications at once. and every time I dismissed that notification, a few more came in.

**Comment by MartinezCatriel** [comment]
Hi! I'm running the same 2 issues. 
Is very annoying, I had to turn off the app 😭 
I'm using Akku on a MacBook air pro 2017 13inch with MacOS Catalina 10.15.7. 
Shouldn't you implement something like [this](https://stackoverflow.com/a/34467770/3088387) in [here](https://github.com/jariz/Akku/blob/9a7ec937f4c9fdf93ace681be562dc9a8b0ae1f4/Akku/StatusMenuController.swift#L110)? along side a behavior like "don't show the notification again after the user has closed it", or at least wait for 3 minutes to show the notification again.

**Comment by SinisterStairs** [comment]
I'm having the same issue with Mojave 10.14.6 (mid-2015 MBP Retina).  Even worse, the spam continues even after I've turned my headset off 5 minutes ago (Sony WH-1000XM4).

When I turned off Bluetooth in macOS, the "Headset will run out of battery soon" spam continued for a a few more seconds before stopping.  As soon as I turned Bluetooth back on (headset is still powered off/not connected), I'm immediately getting the spam again.
